helm/prometheus-rules/templates/platform/atlas/recording-rules/grafana-cloud.rules.yml

```
@@ -327,7 +327,7 @@ spec:
   - name: falco.grafana-cloud.recording
     rules:
     # Falco event counts
-    - expr: sum(falco_events{priority=~"0|1|2|3|4|5|6|7"}) by (cluster_id, cluster_type, customer, installation, pipeline, provider, region, priority, rule)
+    - expr: sum(falcosecurity_falco_rules_matches_total{priority=~"0|1|2|3|4|5|6|7"}) by (cluster_id, cluster_type, customer, installation, pipeline, provider, region, priority, rule_name)
```

So you should be able to use this instead
```suggestion
    - expr: sum(label_replace(falco_events{priority=~"0|1|2|3|4|5|6|7"},"rule_name", "$1", "rule","(.*)") or falcosecurity_falco_rules_matches_total{priority=~"0|1|2|3|4|5|6|7"}) by (cluster_id, cluster_type, customer, installation, pipeline, provider, region, priority, rule_name)
    ```

---

I would suggest that you try it. It works on alba but it only has the old metrics

---

This works but I will tweak it a bit and invert the label replace so I don't have to update the dashboard 😄